src/Components/SubwayNav/SubwayNav.tsx

```diff
@@ -19,6 +19,8 @@ interface OwnProps {
   settingUpIcon: JSX.Element
   settingUpText: string
   setupTime?: string
+  settingUpHeader?: string
+  hideCheckmark?: boolean
```

Comment:
I think we should consider renaming this to `showCheckMark` so that we don't introduce double negatives with `!hideCheckMark`.

That would change the code calling `SubwayNavStep` to something like

```ts
text={value.name}
showCheckmark={this.props.showCheckmark !== false}
```

and the code in `SubwayNavStep` to something like

```ts
stepIsComplete && showCheckmark 
```

what do you think?

Reply:
makes sense to me, i've updated it :)